internal/namespaces/config/commands.go

```diff
@@ -695,6 +696,43 @@ func configImportCommand() *core.Command {
 	}
 }
 
+// configValidateCommand validates the config
+func configValidateCommand() *core.Command {
+	type configValidateArgs struct{}
+
+	return &core.Command{
+		Short:                `Validate the config`,
+		Namespace:            "config",
+		Resource:             "validate",
+		AllowAnonymousClient: true,
+		ArgsType:             reflect.TypeOf(configValidateArgs{}),
+		Run: func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+			configPath := core.ExtractConfigPath(ctx)
+			config, err := scw.LoadConfigFromPath(configPath)
+			if err != nil {
+				return nil, err
+			}
+
+			// validate default profile
+			err = validateProfile(&config.Profile)
+			if err != nil {
+				return nil, err
+			}
+			// validate the remaining profiles
+			for _, profile := range config.Profiles {
+				err = validateProfile(profile)
+				if err != nil {
+					return nil, err
+				}
+			}
+
+			return &core.SuccessResult{
+				Message: "successfully validate config",
```

Comment:
```suggestion
				Message: "configuration is valid",
```

Reply:
It follows the same pattern as other config commands, we may think to rephrase theses messages